evm_arithmetization/README.md

```diff
@@ -10,7 +10,8 @@ Furthermore the implementation itself is highly optimized to provide fast provin
 
 ## Ethereum Compatibility
 
-The aim of this module is to initially provide full ethereum compatibility. Today, all [EVM tests](https://github.com/0xPolygonZero/evm-tests) for the Shanghai hardfork are implemented. Work is progressing on supporting the upcoming [Cancun](https://github.com/0xPolygonZero/plonky2/labels/cancun) EVM changes. Furthermore, this prover uses the full ethereum state tree and hashing modes.
```

Comment:
I've removed the mention to `evm-tests` as they (again) broke the tests upstream (for both Shanghai and Cancun).